src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacade.kt

```diff
@@ -47,42 +47,80 @@ class BookingFacade(
+            NewCourtBookingPrisonCourtEmail(
+              address = contact.email!!,
+              prisonerFirstName = prisoner.firstName,
+              prisonerLastName = prisoner.lastName,
+              prisonerNumber = prisoner.prisonerNumber,
+              court = booking.court!!.description,
+              courtEmailAddress = primaryCourtContact.email!!,
+              prison = prison.name,
+              date = main.appointmentDate,
+              preAppointmentInfo = pre?.let { "${it.startTime.toIsoTime()} to ${it.endTime.toIsoTime()}" },
+              mainAppointmentInfo = main.let { "${it.startTime.toIsoTime()} to ${it.endTime.toIsoTime()}" },
+              postAppointmentInfo = post?.let { "${it.startTime.toIsoTime()} to ${it.endTime.toIsoTime()}" },
+              comments = booking.comments,
+            )
+          } else {
+            NewCourtBookingPrisonNoCourtEmail(
+              address = contact.email!!,
+              prisonerFirstName = prisoner.firstName,
+              prisonerLastName = prisoner.lastName,
+              prisonerNumber = prisoner.prisonerNumber,
+              court = booking.court!!.description,
+              prison = prison.name,
+              date = main.appointmentDate,
+              preAppointmentInfo = pre?.let { "${it.startTime.toIsoTime()} to ${it.endTime.toIsoTime()}" },
+              mainAppointmentInfo = main.let { "${it.startTime.toIsoTime()} to ${it.endTime.toIsoTime()}" },
+              postAppointmentInfo = post?.let { "${it.startTime.toIsoTime()} to ${it.endTime.toIsoTime()}" },
+              comments = booking.comments,
+            )
+          }
         }
-      }.forEach { email ->
-        emailService.send(email).onSuccess { (govNotifyId, templateId) ->
-          notificationRepository.saveAndFlush(
-            Notification(
-              videoBooking = booking,
-              email = email.address,
-              govNotifyNotificationId = govNotifyId,
-              templateName = templateId,
-              reason = "New court booking request",
-            ),
-          )
-        }.onFailure { log.info("BOOKINGS: Failed to send new court booking email.") }
+
+        else -> log.info("No contacts found for video booking ID ${booking.videoBookingId}").let { null }
       }
+    }.forEach { email ->
+      emailService.send(email).onSuccess { (govNotifyId, templateId) ->
+        notificationRepository.saveAndFlush(
+          Notification(
+            videoBooking = booking,
+            email = email.address,
+            govNotifyNotificationId = govNotifyId,
+            templateName = templateId,
+            reason = "New court booking request",
+          ),
+        )
+      }.onFailure { log.info("BOOKINGS: Failed to send new court booking email.") }
+    }
   }
 
+  private fun Collection<BookingContact>.primaryCourtContact() =
+    allContactsWithAnEmailAddress().singleOrNull { it.contactType == ContactType.COURT && it.primaryContact }
```

Comment:
I did wonder about expanding the BookingContactsService to provide such things but thought until we need it elsewhere then perhaps no real need as of yet.